aws/istio-ingress/base/ingress.yaml

```diff
@@ -1,16 +1,18 @@
-apiVersion: extensions/v1beta1 # networking.k8s.io/v1beta1
+apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   annotations:
-    kubernetes.io/ingress.class: alb
-    alb.ingress.kubernetes.io/scheme: $(loadBalancerScheme)
     alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}]'
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    kubernetes.io/ingress.class: alb
+  labels:
```

Comment:
remove the labels. This is generated after your run `kustomize build`.  You should minimize the changes in this file. 
Just update the line without changing the sequence.

Reply:
Removed labels in latest commit